cmsdev/internal/lib/common/cmd.go

```diff
@@ -102,6 +102,28 @@ func (cmdResult *CommandResult) SetEnvVars() string {
 	return envVarNames.String()
 }
 
+// RunWithRetry executes the command and retries if output contains "503 Service Unavailable".
+// maxRetries specifies how many times to retry (not counting the first attempt).
+// retryDelay specifies the delay between retries.
+func (cmdResult *CommandResult) RunWithRetry() (err error) {
+	maxRetries := 3
+	retryDelay := 5 * time.Second
+	for attempt := 0; attempt < maxRetries; attempt++ {
+		err = cmdResult.Run()
+		if strings.Contains(cmdResult.OutString(), "503 Service Unavailable") ||
```

Comment:
In cases like these, where the "else" part of the conditional always ends with a return statement. I suggest refactoring so that the "else" becomes the "then" part, so that you then no longer need the "else" at all, and there is less indentation.

What I mean is, in general, rather than:

```
if whatever, then
    blah1
    blah2
    blah3
else
    harf1
    harf2
    harf3
    return
```

Unless there is a good reason not to, I would refactor that to

```
if not whatever, then
    harf1
    harf2
    harf3
    return
blah1
blah2
blah3
```